Assets/SteamVR_Unity_Toolkit/Scripts/Controls/3D/VRTK_Control.cs

```diff
@@ -72,7 +92,7 @@ public virtual void Update()
                 if (value != oldValue)
                 {
                     HandleInteractables();
-                    defaultEvents.OnValueChanged.Invoke(getValue());
+                    defaultEvents.OnValueChanged.Invoke(getValue(), getNormalizedValue());
```

Comment:
Is this an event we should document in all of the other inherited classes as I'm guessing they all throw this event?